proto.lock

```diff
@@ -143,13 +143,7 @@
         ],
         "package": {
           "name": "node"
-        },
```

Comment:
you need exact version of protolock to avoid additional changes in proto.lock (`github.com/nilslice/protolock/cmd/protolock@v0.12.0`)